### DISCUSSION TITLE
دادن لینک اشتراک عادی در صورت فعال کردن حالت پیش‌فرض "لینک اشتراک جیسون"

### DISCUSSION THREAD

**Discussion by farshadl** [Gozargah/Marzban#1120] [General]
درود.
دوستان من توی تنظیمات مرزبان گزینه زیر را فعال کردم (حالت پیش‌فرض اشتراک را گذاشتم روی JSON).
USE_CUSTOM_JSON_DEFAULT=True
اگه بخوام استثنا به یکی لینک استراک عادی بدم، امکان‌پذیر هست؟

**Comment by Mjavadkhoini** [comment]
سلام

اره میشه
اگه جیسون رو فعال کردی و میخوای لینک عادی بدی کافیه که یه
/v2ray
آخر لینک ساب اون کاربر قرار بدی

 اگر لینک ساب عادی هستش و میخوای به یه کاربر جیسون بدی کافیه که
/v2ray-json
آخرش اضافه کنی

**Reply by sirvan1133** [reply]
کاربلد ❤️
